sql/job_points.sql

```diff
@@ -0,0 +1,102 @@
+INSERT INTO `job_points` VALUES ('268', 'enmity_douse_recast', '2', '4');
+INSERT INTO `job_points` VALUES ('270', 'manawell_effect', '1', '4');
+INSERT INTO `job_points` VALUES ('272', 'magic_burst_enmity_decrease', '1', '4');
+INSERT INTO `job_points` VALUES ('274', 'magic_damage_bonus', '1', '4');
+INSERT INTO `job_points` VALUES ('320', 'chainspell_effect', '2', '5');
+INSERT INTO `job_points` VALUES ('322', 'convert_effect', '1', '5');
+INSERT INTO `job_points` VALUES ('324', 'stymie_effect', '1', '5');
+INSERT INTO `job_points` VALUES ('326', 'magic_accuracy_bonus', '1', '5');
+INSERT INTO `job_points` VALUES ('328', 'composure_effect', '1', '5');
+INSERT INTO `job_points` VALUES ('330', 'magic_atk_bonus', '1', '5');
+INSERT INTO `job_points` VALUES ('332', 'saboteur_effect', '2', '5');
+INSERT INTO `job_points` VALUES ('334', 'enfeebling_magic_duration', '1', '5');
+INSERT INTO `job_points` VALUES ('336', 'quick_magic_effect', '2', '5');
+INSERT INTO `job_points` VALUES ('338', 'enhancing_magic_duration', '1', '5');
+INSERT INTO `job_points` VALUES ('384', 'perfect_dodge_effect', '3', '6');
+INSERT INTO `job_points` VALUES ('386', 'sneak_attack_effect', '1', '6');
+INSERT INTO `job_points` VALUES ('388', 'larceny_duration', '1', '6');
+INSERT INTO `job_points` VALUES ('390', 'trick_attack_effect', '1', '6');
+INSERT INTO `job_points` VALUES ('392', 'steal_recast', '2', '6');
+INSERT INTO `job_points` VALUES ('394', 'mug_effect', '5', '6');
+INSERT INTO `job_points` VALUES ('396', 'despoil_effect', '2', '6');
+INSERT INTO `job_points` VALUES ('398', 'conspirator_effect', '1', '6');
+INSERT INTO `job_points` VALUES ('400', 'bully_effect', '1', '6');
+INSERT INTO `job_points` VALUES ('402', 'triple_attack_effect', '1', '6');
+INSERT INTO `job_points` VALUES ('448', 'invincible_effect', '100', '7');
+INSERT INTO `job_points` VALUES ('450', 'holy_circle_effect', '1', '7');
+INSERT INTO `job_points` VALUES ('452', 'intervene_effect', '2', '7');
+INSERT INTO `job_points` VALUES ('454', 'sentinel_effect', '1', '7');
+INSERT INTO `job_points` VALUES ('456', 'shield_bash_effect', '10', '7');
+INSERT INTO `job_points` VALUES ('458', 'cover_duration', '1', '7');
+INSERT INTO `job_points` VALUES ('460', 'divine_emblem_effect', '2', '7');
+INSERT INTO `job_points` VALUES ('462', 'sepulcher_duration', '1', '7');
+INSERT INTO `job_points` VALUES ('464', 'palisade_effect', '1', '7');
+INSERT INTO `job_points` VALUES ('466', 'enlight_effect', '1', '7');
+INSERT INTO `job_points` VALUES ('512', 'blood_weapon_effect', '2', '8');
+INSERT INTO `job_points` VALUES ('514', 'arcane_circle_effect', '1', '8');
+INSERT INTO `job_points` VALUES ('516', 'soul_enslavement_effect', '1', '8');
+INSERT INTO `job_points` VALUES ('518', 'last_resort_effect', '2', '8');
+INSERT INTO `job_points` VALUES ('520', 'souleater_duration', '1', '8');
+INSERT INTO `job_points` VALUES ('522', 'weapon_bash_effect, '1', '8');
```

Comment:
@bendangelo technically all these things shouldn't be single quoted to begin with, other than this one field he missed that on (because strings). Looks like it was dumped from old-ass navicat, which single quotes anything and everything whether it needs it or not.

...It'll still work even though its funky lookin though, as long as he put that missing one in.